content/applications/hr/payroll.rst

```diff
@@ -311,49 +445,86 @@ The required fields for an advantage are:
   :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
 - :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
   advantage applies to.
+- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+
+.. image:: payroll/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
 
 Personal info
 -------------
 
-Every employee in Odoo has an *employee card* that includes all of their personal information,
-resume, work information, and documents. To view an employee's card, go to the main
-:menuselection:`Payroll` app dashboard, and click on the employee's card, or go to
-:menuselection:`Payroll --> Employees --> Employees` and click on the employee's card. Employee
-cards can also be viewed by going to the :menuselection:`Employees` app.
+Every employee in Odoo has an *employee card* which is created when a candidate becomes an
+employee. This card includes all of their personal information, resume, work information, and
+documents.
+
+The personal information is gathered from the salary package configurator section that a
+candidate fills out after being offered a position. This personal information is then transferred to
+the employee card when they are hired.
+
+To view an employee's card, go to the main :menuselection:`Employees` app dashboard, and click on
+the employee's card.
 
 .. note::
    An employee card can be thought of as an employee personnel file.
 
 The *Personal Information* section lists all of the fields that are available to enter on the
-employee's card. To access this section, go to :menuselection:`Payroll --> Configuration -->
-Personal Info`.
+employee's card. To access this section, go to :menuselection:`Payroll --> Configuration --> Salary
+Package Configurator  --> Personal Info`.
 
 .. image:: payroll/personal-info.png
    :align: center
-   :alt: Personal information that appear on employee cards to enter.
+   :alt: A list of all the personal information that appears on employee card to enter.
 
-To edit an entry, select it from the list. Then, click the :guilabel:`Edit` button, and modify the
-entry. When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel
-the edits.
+To edit a personal info entry, select the entry from the list, and modify the personal info. To
+create a new personal info entry, click the :guilabel:`New` button.
 
-.. image:: payroll/personal-new.png
-   :align: center
-   :alt: New personal information entry.
+The required fields, aside from entering the :guilabel:`Information` name, are
+:guilabel:`Related Field` and :guilabel:`Category`. Select a :guilabel:`Related Field` from the
```

Comment:
Wow - I checked this several times, in runbots in 16-16.4, and for whatever reason I missed this! Also, I can't NOT select one, so that is also why I missed this. I try to delete everything, click save, to see what's required- but thisfield is not possible to leave blank. Very good catch- thank you!